src/components/Races/RaceTableRow.tsx

```diff
@@ -6,31 +7,59 @@ import utils from "../../utils";
 type Props = {
   runner?: Runner;
   onClick: (runner?: Runner) => void;
+  isScratched?: boolean;
 };
 
-export const RaceTableRow: React.FC<Props> = ({ runner, onClick }) => {
+export const RaceTableRow: React.FC<Props> = ({
+  runner,
+  onClick,
+  isScratched
+}) => {
   const { number, name, barrier, odds, handicapWeight, last5Starts } =
     runner || {};
 
   return (
     <tr
-      className="cursor-pointer hover:bg-gray-100"
-      onClick={() => onClick(runner)}
+      className={classnames({
+        "cursor-pointer hover:bg-gray-100": !isScratched,
+        "cursor-not-allowed": isScratched
```

Comment:
`cursor-not-allowed` might look a bit ugly -- could probably just use `cursor-default`